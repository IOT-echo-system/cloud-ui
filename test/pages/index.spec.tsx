import {render} from '@testing-library/react'
import HomePage from '../../src/pages'

describe('Home page test', () => {
  it('should match with the snapshot', () => {
    const {container} = render(<HomePage />)

    expect(container).toMatchSnapshot()
  })
})
