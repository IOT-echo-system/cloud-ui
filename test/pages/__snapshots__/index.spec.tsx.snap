// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page test should match with the snapshot 1`] = `
<div>
  <div
    class="MuiStack-root css-nen11g-MuiStack-root"
  >
    Home Page
  </div>
</div>
`;
